otwcode/otwarchive
# AO3-4608

## Issue

https://otwarchive.atlassian.net/browse/AO3-4608

## Purpose

This PR fixes a problem whereby the URL that initiated the Abuse Report was not being forwarded on to the Abuse and Support ticket management site. 

## Testing

In order to test:

1. Submit a ticket to the Abuse ticket management site via the test Archive (http://test.archiveofourown.org/abuse_reports/new). 

2. Check the Abuse ticket management site and look for your ticket. Once you've found your ticket, ensure that the URL field has some content. The URL field is located on the right hand side of the ticket with all of the other fields. 



## References

None.

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](http://archiveofourown.org/admin_posts?tag=1)?*

Scott

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

He/him
